datadog_csi_driver/assets/configuration/spec.yaml

```
@@ -0,0 +1,10 @@
+name: datadog_csi_driver
+files:
+- name: datadog_csi_driver.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

```suggestion
    - template: instances/openmetrics
```
You'll need to sync the config with: `ddev validate config datadog_csi_driver -s`